### PR TITLE
Create Youtube Video Viewing Homepage.

### PR DESCRIPTION
**Pull request by cod3smith**
#### What does this PR do?
- This PR contains the code for creating a simple Youtube video homepage. It uses **CSS Flexbox** and **CSS Grid** to accomplish this. It also demonstrates the use of semantic HTML tags from **HTML5**

#### How would you run this project?
- Clone the project.
- Run the `index.html` file on your browser.

### CODE REVIEW

**Comment by Dandush03** [comment]
# :red_circle: Status Require: changes :muscle: 
### Hey :wave: 
## Good attempts on this one :muscle: :+1: 

## Here are the issues to address to get the project meet the requirement:wink:

# Description

## Project Specification
Don't forget that this project consists of building an HTML document that matches the appearance of this [YouTube video player page](http://archive.vn/Bss88).

Check it [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/)

-------------------------------------------

### Page structure

-------------------------------------------
#### Navigation bar

##### Left Part
- [x] Logo is too small
- [x] Hamburger button is too big

##### Center
- [x] Input should be wider
- [x] Search button missing padding
- [x] Search Button missing same border as input
- [x] Search Button need a more clear background

##### Right
- [x] Buttons should be aligned to the right
- [x] There should be only an upload button and a blue sign in one

-------------------------------------------

#### Require:
Screenshot from 2020-05-07 08-14-35

#### Yours:
Screenshot from 2020-05-07 08-14-00

-------------------------------------------

#### Video

##### Title (Aligned to the Left)

- [x] It should contain a hero profile picture
- [x] It should contain a hero name
- [x] It should contain a subscribe button
- [x] It should contain the number of views (aligned to the right)

##### Title Buttons

- [x] Missing "Add to", "Share", "More", "Like" and "Dislike" buttons

##### Description

- [x] It should contain the published date

-------------------------------------------

#### Require:
Screenshot from 2020-05-07 08-20-45

#### Yours:
Screenshot from 2020-05-07 08-21-00

-------------------------------------------

### Sidebar Recommended videos

- [x] Missing Titles ("Up Next" and "Autoplay")
- [x] Description is too big
- [x] It Should show the numbers of views

-------------------------------------------

#### Require:
Screenshot from 2020-05-07 08-28-38

#### Yours:
Screenshot from 2020-05-07 08-28-54

-------------------------------------------

### Additional Microverse Requirements (Pathwright)

#### General

- [x] Check HTML markup with this [validator](https://validator.w3.org/).([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)

-------------------------------------------

#### Pic:
Screenshot from 2020-05-07 08-06-52



## Kindly address all these changes and submit a new code review


### Happy coding:computer: 

#### Daniel Laloush

**Comment by berhanu-tarekegn** [comment]
## Updated PR
- We updated the project based on the code review comments given previously.
- Validated the **CSS** and the **HTML** using [W3 Validator](https://validator.w3.org/)
- We managed to add the readme section based on **microverse** template 

@Dandush03 a kind reminder here, could you please take a look at.

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    height: 72px;
+}
+.left {
+    display: flex;
+    align-items: center;
+}
+.center {
+    width: 100%;
+    display: flex;
+    align-items: center;
```
Comment (rammazzoti2000) [inline assets/css/style.css]: CSS bad practice: no repeating of the same rule
```
.left,
.right,
.share {
    display: flex;
    align-items: center;
}
.center,
.current_video,
.comments {
    width: 100%;
}
.share {
    margin: auto
}
```

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+
+.description > div {
+    display: flex;
+    flex-direction: column;
+    width: 80%;
+}
+.description > div > p {
+    padding-bottom: 20px;
+}
+.comments {
```
Comment (rammazzoti2000) [inline assets/css/style.css:117]: CSS bad practice: no repeating of the same rule. See first inline comment.

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+    display: flex;
+    justify-content: space-between;
+    padding: 10px 10px;
+    border-top: 2px #ccc solid;
+    border-bottom: 2px #ccc solid;
+    margin-bottom: 5px;
+    background: white;
+    margin-bottom: 20px;
+}
+.share {
```
Comment (rammazzoti2000) [inline assets/css/style.css:247]: CSS bad practice: no repeating of the same rule, see the first inline comment

File: index.html [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+    <header>
+        <nav class="nav_menu">
+            <div class="left">
+                <img src="./assets/img/menu.png" alt="Menu">
+                <img src="./assets/img/youtube.png" id="youtube-logo" alt="Logo">
+            </div>
+            <div class="center">
+              
+                <input id="search-input" type="text" placeholder="Search something...">
+                <img src="./assets/img/search.png" alt="" id="search-btn">
```
Comment (rammazzoti2000) [inline index.html]: Should be a `<button>` tag that represents the search button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/search_button_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Hello @KelynPNjeri and @berhanu-tarekegn :wave:  great job this far :100:
Great job and implementing the changes required by the previous  TSE
There are still though some issues that need to be fixed before approving this project :pray: 
### :stop_sign: Require Changes :construction: :construction_worker: 
- [x] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright), in this case no repeating of the same rule, the following has been refactored to avoid repeating the same rule many times :point_down:, please see the inline comment
```
.left,
.right,
.share {
    display: flex;
    align-items: center;
}
.center,
.current_video,
.comments {
    width: 100%;
}
.share {
    margin: auto
}
```
- [ ] The button to the right of the input must be a `<button>` tag that represents the search button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/search_button_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/)), yours is an image
- [ ] The NavBar It has at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/icons_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
#### Clone: 
image
#### Original:
image
- [ ] Make the number of subscribers is aligned on the oh the subscribe button and the like/dislike buttons should be better aligned they seem to be a bit off
image
- [x] Also make sure to properly align you like/dislike and reply button in you comments section, and the picture must be render correctly, yours is stretched out.
#### Clone:
image
#### Original: 
image
### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

##### Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>

### :stop_sign:  Status: Changes required.

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### :stop_sign:  Status: Required Changes

**Comment by cod3smith** [comment]
## Updated PR
- [x] We updated the project based on the code review comments given previously.
- [x] We fixed the alignment issues that were present on the navbar and on the sidebar. 
- [x] We also added more icons to the right div on the navbar.
- [x] We aligned the like and dislike buttons together with text accompanying it.

**Comment by santiago-dev96** [comment]
# Pull Request under review by @santiago-rodrig (TSE)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+                
+                <div class="channel-details">
+                    <img id="profile" src="./assets/img/account_box-24px.svg" alt="Profile Picture">
+                
+                    <div class="channel-actions">
+                        <p>The odin project</p>
+                        <div></div>
+                        
+                        <span class="subscribers"><button id="subscribe-btn">Subscribe</button>
+                            <div class="number_of_subscribers">799</div>
```
Comment (santiago-dev96) [inline index.html]: This caused a validation error

**Review by santiago-dev96 — APPROVED** [review]
# Project approved :100:
Good job implementing the previous TSE required changes, you meet with all the requirements for approval, you can go on with the next projects now.

Have a nice day :wave:!

**Pull request merged**